lib/components/dropdown.vue

```diff
@@ -43,15 +43,6 @@
     </div>
 </template>
 
-<style scoped>
-    .dropdown-item:focus,
-    .dropdown-item:hover,
-    .dropdown-header:focus {
-        background-color: #eaeaea;
```

Comment:
This line was to make focus/hover state more "obvious" since the current bootstrap css the hover state is very very subtle (and hard to tell what is focused for keyboard users, since the default hover color is almost white)

Maybe I should open an issue on Bootstrap about this.